### ISSUE TITLE
Wound Animation Loops Infinitely

### ISSUE THREAD

**Issue by violetcrestedwren** [Lurkars/gloomhavensecretariat#95]
I've seen this bug multiple times with the Animations enabled (the `Disable Animations` tag is _not_ selected) and `Automatic apply conditions` _is_ selected. It seems to occur during times when the app tries to end the wounded monster's turn while the animation is still occurring. The animation continues to loop indefinitely, and most inputs cause the server to hang. I haven't seen the issue occur after toggling those two settings. Encountered in Jaws of the Lion & Gloomhaven, haven't tested with Regeneration.

**Comment by Lurkars** [comment]
Any hints too reproduce this? Was this on server connection with multiple clients?

**Comment by violetcrestedwren** [comment]
> Any hints too reproduce this? Was this on server connection with multiple clients?

I can't say for sure that this will reproduce it reliably, but getting going back to the start of the round by accidentally clicking the first character in initiative after completing other actions has definitely caused the issue (assuming 1+ enemy with wound is somewhere in the middle of the initiative order - not first or last). My group has three characters each connected on separate accounts. It's happened on a privately hosted server as well as the public one (we initially assumed the cheap NAS hosting the server was having resource issues, so we switched to the public server). Looking for ways to reproduce reliably now.

**Comment by violetcrestedwren** [comment]
Public server instance that appears to be in the broken state: "woundisbroken".

**Comment by violetcrestedwren** [comment]
I created a scenario with three characters, spawned several enemies, and wounded most of them. I then set initiatives of the character to be early in the round, middle, and late, respectively. After starting the round I clicked through initiative orders, draw monster cards, added status effects, adjusted hp, just randomly and rapidly altered the state of the app. When only one user was connected, this all worked fine - if I went very rapidly it sometimes had a delay as all the animations played out before everything caught up, but eventually it got there successfully. I could not trigger the bugged state with only one user connection.

I then connected a second user in a separate browser and started clicking around again and very quickly entered the bugged state.
